shift_detector/precalculations/count_vectorizer.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+from shift_detector.precalculations.precalculation import Precalculation
+from nltk.corpus import stopwords as nltk_stopwords
+from sklearn.feature_extraction.text import CountVectorizer as CountVectorizer_sklearn
+from shift_detector.utils.column_management import ColumnType
+
+
+class CountVectorizer(Precalculation):
+
+    def __init__(self, cols, stop_words='english', max_features=None):
```

Comment:
The parameter should definitely include the word 'language' in it's name. Maybe stop_word_language?

Reply:
I would argue against that. Gensim and nltk for example also uses the stop_words parameter exactly the same way it is used here. Hence for better understanding, it is called the same.